### ISSUE TITLE
Cannot Get this plugin to work.

### ISSUE THREAD

**Issue by umsdavid** [bitpay/cordova-plugin-qrscanner#217]
Hi,

Is there any example on how to put the script in index.html and index.js? I've tried many times but the apps doesnt seems to work

**Comment by Junijoh** [comment]
Yes this doesnt seem to work, The guide described in this plugin is not helpful and seems very vague.

**Comment by Junijoh** [comment]
I would like to know of a qr code  + barcode scanner plugin for cordova 8.1.2

**Comment by irvine48** [comment]
so.. this plugin seen not maintain anymore? i tested "phonegap-plugin-barcodescanner" just work fine. But not this which i tested first in the place. Still cant make it work. Camera no trigger.

**Comment by Junijoh** [comment]
Yes so can anyone please tell what we can if this is discontinued.
Do we have any other alternative?

**Comment by irvine48** [comment]
> Yes so can anyone please tell what we can if this is discontinued.
> Do we have any other alternative?

as mentioned, phonegap-plugin-barcodescanner work fine for me. here attached with the example,
https://www.techiediaries.com/barcode-qr-code-scanner-encoder-ionic-3/

**Comment by Junijoh** [comment]
> > Yes so can anyone please tell what we can if this is discontinued.
> > Do we have any other alternative?
> 
> as mentioned, phonegap-plugin-barcodescanner work fine for me. here attached with the example,
> https://www.techiediaries.com/barcode-qr-code-scanner-encoder-ionic-3/

Yes phonegap-plugin-barcodescanner works.Thanks! Is there a way we can set the camera quality while scanning the barcode/ qrcode?
